homeassistant/components/light/osramlightify.py

```diff
@@ -198,6 +196,8 @@ def turn_on(self, **kwargs):
             self._luminary.set_luminance(
                 int(self._brightness / 2.55),
                 transition)
+        else:
+            self._luminary.set_onoff(1)
```

Comment:
I checked your point and you are right. Colour and temperature cannot be changed with this new implementation starting from a light off state. I will try to adapt my implementation in order to take care of this scenario.